lib/ngx/ssl.md

```diff
@@ -308,6 +312,68 @@ This function can be called in whatever contexts where downstream https is used.
 
 [Back to TOC](#table-of-contents)
 
+parse_pem_cert
+----------------
+**syntax:** *cert_chain, err = ssl.parse_pem_cert(pem_cert_chain)*
+
+**context:** *any*
+
+Converts the PEM-formated SSL certificate chain data into an opaque cdata pointer (for later uses
+in the [set_cert](#set_cert)
+function, for example).
+
+In case of failures, returns `nil` and a string describing the error.
+
+You can always use libraries like [lua-resty-lrucache](https://github.com/openresty/lua-resty-lrucache#readme)
+and/or ngx_lua APIs like [lua_shared_dict](https://github.com/openresty/lua-nginx-module#lua_shared_dict)
+to do the caching of the cdata results, for example.
```

Comment:
I don't think lua_shared_dict can cache the cdata returned directly. We need some kind of serialization here. Thoughts?